BetterReads/BetterReads/Models/Book.swift

```diff
@@ -56,6 +56,21 @@ struct SearchResult: Codable {
     let totalItems: Int
 }
 
+struct RecommendationsResult: Codable {
```

Comment:
Cut out lines 59 through 73 in this file (so the closing brace on line 57 and the // MARK: - Item comment on line 74 only have 1  space between them again) and put the RecommendationsResult and Recommendation structs into a new model file because I've already changed this file so we're going to have a !mErGe CoNfLiCt!_(&%#(_#&